src/flow_solver/flow_solver_cases/flow_solver_case_base.cpp

```diff
@@ -82,6 +82,14 @@ double FlowSolverCaseBase<dim,nstate>::get_constant_time_step(std::shared_ptr<DG
     return all_param.ode_solver_param.initial_time_step;
 }
 
+template <int dim, int nstate>
+double FlowSolverCaseBase<dim,nstate>::get_adaptive_time_step(std::shared_ptr<DGBase<dim,double>> /*dg*/) const
+{
+    pcout << "Using initial time step in ODE parameters." <<std::endl;
+    pcout << " Base definition for get_adaptive_time_step() has not yet been implemented." <<std::endl;
+    return all_param.ode_solver_param.initial_time_step;
```

Comment:
Maybe a better behaviour would be to abort if the user accidentally sets `adaptive_time_step=true` while this isn't implemented. Shouldn't really matter if the intention is to add this soon, though.

Reply:
fixed in most recent commit